docs/guides/README.md

```diff
@@ -997,9 +997,299 @@ go run app.go
+
+### 3. Configure the `pom.xml` file to manage the dependencies
+We now need to add the following dependencies into maven’s configuration file. Copy the below content into your `pom.xml` file.
+```xml
+#pom.xml
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.3.3.RELEASE</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.example</groupId>
+    <artifactId>demo1</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>demo1</name>
+    <description>Demo project for Spring Boot</description>
+
+    <properties>
+        <java.version>1.8</java.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-thymeleaf</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.junit.vintage</groupId>
+                    <artifactId>junit-vintage-engine</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-maven-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
+</project>
+````
+Right-Click on `pom.xml` file and Select (Maven -> Reimport) to import all the dependencies in your project.
+
+### 4. Adding Cosmic Credentials to Java Spring Boot Project
+Inside the folder src/main/resources, add the cosmic credentials to the `application.properties` file.
+```properties
+#src/main/resources/application.properties
+slug = <add cosmic bucket slug here>
+read_key = <add cosmic bucket read key here>
+
+```
+
+### 5. Create the service component of Java Spring Boot App
+In the folder `src/main/java`, Add a new Java package (folder) `service` inside the package `com.example.cosmicapp`.
+This can be done by right-clicking on the package `com.example.cosmicapp`, selecting (New -> Package), and editing `com.example.cosmicapp.service` in the field.
```

Comment:
All this can be little simplified like:

```Inside the folder `src/main/java`, right-click on `com.example.cosmicapp` package and add a new package named `service` by selecting (New -> Package).```

This schema should be followed in all the cases below as well.